lightspeed/modules/proc_login-vscode-extension.adoc

```diff
@@ -5,7 +5,34 @@
 
 After installing and configuring the VS Code extension, you can log in to the Ansible Lightspeed service.
 
-.Procedure
+== Sign-in options
+
+{LightspeedShortName} provides different sign-in methods depending on whether you are using the cloud service or the on-premise deployment.
+
+* *Ansible Lightspeed on-premise deployments*
++
+Users are authenticated using your {PlatformName} login. 
++
+To sign in, you can use the *Connect* button or the *Accounts* button, and use the option *Sign in with Ansible Lightspeed to use Ansible* from VS Code. Once prompted, select *Log in with {PlatformNameShort}*, and log in with the authorization mechanism that your automation controller is configured with. 
+
+* *Ansible Lightspeed cloud service*
++
+Users are authenticated using Red Hat Single Sign-On (RH-SSO).
++
+To sign in, you can use the *Connect* button or the *Accounts* button and then select one of the following options:
+
+** *Sign in with Ansible Lightspeed to use Ansible*
+** *Sign in with Red Hat to use Ansible*
++
+Based on the option you selected, follow the on-screen prompts to log in and access the Ansible Lightspeed service.
++
+[NOTE]
++
+====
+If you are using a cloud development environment at a domain unknown by Ansible Lightspeed, such as on-premise Red Hat OpenShift Dev Spaces, your *Accounts* sign-in menu provides the option *Sign-in with Red Hat to use Ansible*. This option uses a device code flow to successfully complete the sign-in process and requires the Red Hat Authentication extension v0.2.0 or later. If you require this authentication flow but don't see the *Sign-in with Red Hat to use Ansible* option, ensure you are using the Ansible VS Code extension v24.5.2 or later.
```

Comment:
Same question as before about using `{RHSSO} `

Reply:
This is the UI label